docs/source/histo_workflow.md

```diff
@@ -0,0 +1,6 @@
+# Histopathology Pre-processing Workflow
+
+1. Montage: Create a montage from the files in the H&E folder. Please follow the [instructions here](montage_creation.md).
```

Comment:
Not necessarily H&E, should we maybe change it by WSI? or is this specific to H&E? What about IHC?
Should we maybe precise that it's H&E workflow?

Reply:
True, made this more generic now